### PR TITLE
incorporating review comments

### PR DESCRIPTION
**Pull request by rkedar1**

### CODE REVIEW

File: doc/documents/MLI_kernels/comm_lstm.rst [hunk 1]
```diff
@@ -205,37 +192,45 @@ Where:
 Function Configuration Structure
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (dzakhar) [inline doc/documents/MLI_kernels/comm_lstm.rst]: Please, move configuration structure description to Basic RNN section and left here only reference to it (at the moment basic rnn cites to the configuration structure, it should be swapped)

File: doc/documents/MLI_kernels/comm_lstm.rst [hunk 2]
```diff
@@ -205,37 +192,45 @@ Where:
...
+Parameters
+''''''''''
+
+.. table:: Function Configuration Parameters
+   :widths: 20,80
+
+   +-----------------------+-----------------------+
+   |  **Fields**           |  **Description**      |
+   +=======================+=======================+
+   | ``mode``              | LSTM processing mode  |
```
Comment (dzakhar) [inline doc/documents/MLI_kernels/comm_lstm.rst]: Replace LSTM -> RNN

File: doc/documents/MLI_kernels/comm_lstm.rst [hunk 3]
```diff
@@ -205,37 +192,45 @@ Where:
...
+   |                       | (enumeration)         |
+   +-----------------------+-----------------------+
+   | ``act``               | LSTM output           |
```
Comment (dzakhar) [inline doc/documents/MLI_kernels/comm_lstm.rst]: Replace LSTM -> RNN

File: doc/documents/MLI_kernels/convolution_2d.rst [hunk 4]
```diff
@@ -74,182 +75,202 @@ Convolution 2D
...
+	||                           *CHW Data Layout*                            |
+	+-------------------------------------+-----------------------------------+
+	| ``mli_krn_conv2d_chw_fx8``          | Switching function; 8bit FX       |
+	|                                     | tensors; Delegates calculations   |
+	|                                     | to suitable specialization or     |
+	|                                     | generic function.                 |
+	+-------------------------------------+-----------------------------------+
+	| ``mli_krn_conv2d_chw_fx16``         | Switching function; 16bit FX      |
+	|                                     | tensors;                          |
+	|                                     |                                   |
```
[the remark this answers is not in the document]
Comment (dzakhar) [reply doc/documents/MLI_kernels/convolution_2d.rst]: *changes font size, sorry. The same for the other similar comments

File: doc/documents/conf.py [hunk 5]
```diff
@@ -86,7 +86,7 @@
 todo_include_todos = True
 
 #If true, figures and tables are numbered
-numfig = True
+numfig = False
```
Comment (ipozdnov) [inline doc/documents/conf.py]: This is comment to line 66 above.
Change version = u'1.00a' to version = u'1.00'

File: doc/documents/getting_started/getting_started.rst [hunk 6]
```diff
@@ -53,21 +53,21 @@ Build Library
```
[the remark this answers is not in the document]
Comment (demidan) [reply doc/documents/getting_started/getting_started.rst]: I've discussed this with Kedar, he will add "command prompt" ('$ ' for Linux and '> ' for Windows examples) before commands, and leave commands output w/o "prompt".
The same formatting is used in other embarc projects (e.g. https://embarc.org/toolchain/baremetal/em-starter-kit.html)

File: doc/documents/conf.py (hunk 5 above)
Comment (dzakhar) [reply doc/documents/conf.py]: Done it. Please check

File: doc/documents/MLI_FP_data_format/MLI_FP_data_format.rst [hunk 7]
```diff
@@ -90,225 +94,227 @@ Data storage
...
+ - :math:`\ real\_ val\ ` \- real value (might be represented as float)
+ - :math:`\ dequant\_ real\_ val\ ` \- dequantized real value
+ - :math:`\ fx\_ val\ ` \- FX value of the particular Q format
+ - :math:`\ fraq\_ bits \ ` \- number of :math:`\ fx\_ val\ ` fractional bits
+ - :math:`\ Round\ () \ ` \- rounding according to one of supported modes
+   
+2 :sup:`fraq_bits` represents 1.0 in FX format and also might
+be obtained by shifting (1 << fraq_bits). Rounding mode (nearest, up,
+convergence) affects only FX representation accuracy. MLI Library
+uses rounding provided by ARCv2 DSP hardware (see :ref:`hw_comp_dpd` ). ``dequant_real_val`` might be not equal to
```
Comment (demidan) [inline doc/documents/MLI_FP_data_format/MLI_FP_data_format.rst]: 'dequant_real_val' need "math" formatting

File: doc/documents/MLI_FP_data_format/MLI_FP_data_format.rst [hunk 8]
```diff
@@ -90,225 +94,227 @@ Data storage
...
+``real_ val`` in case of immediate forward/backward conversion
```
Comment (demidan) [inline doc/documents/MLI_FP_data_format/MLI_FP_data_format.rst]: 'real_val' need math formatting. Also remove spece between 'real_' and 'val'.

File: doc/documents/MLI_FP_data_format/MLI_FP_data_format.rst [hunk 9]
```diff
@@ -90,225 +94,227 @@ Data storage
...
+.. note::
 
    Examples:
 
-   -  For a real value of 0.85; FX format Q.7; rounding mode nearest, the
-      FX value is computed as:
-
+   -  Given a real value of 0.85; FX format Q.7; rounding mode nearest, the
+      FX value is computed as: 
       ``Round(0.85 * (2^7)) = Round(0.85 * 128) = Round(108.8) = 109 (0x6D)``
```
Comment (demidan) [inline doc/documents/MLI_FP_data_format/MLI_FP_data_format.rst]: Need "math" (not "code") formatting here and in other examples below

File: doc/documents/MLI_kernels/comm_lstm.rst [hunk 10]
```diff
@@ -252,141 +247,153 @@ Function Configuration Structure
...
+
+.. code:: c                               
+                                          
+ mli_status mli_krn_lstm_cell_<data_type> 
+ [_specialization](                       
+    const mli_tensor *in,                 
+    const mli_tensor *prev_out,           
+    const mli_tensor *weights,            
+    const mli_tensor *bias,               
+    const mli_lstm_cell_cfg *cfg,
```
Comment (demidan) [inline doc/documents/MLI_kernels/comm_lstm.rst]: please change `mli_lstm_cell_cfg` to `mli_rnn_cell_cfg`

**Pull request merged**